phone_proxy/lib/main.dart

```diff
@@ -1,149 +1,231 @@
-// ignore_for_file: avoid_print, public_member_api_docs
 import 'dart:async';
 import 'dart:convert';
 import 'dart:typed_data';
 
-import 'package:ble/ble.dart';
-import 'package:ble/ble_central.dart';
-import 'package:ble/ble_peripheral.dart';
-import 'package:ble/ble_socket.dart';
+import 'package:blev/ble.dart';
+import 'package:blev/ble_central.dart';
+import 'package:blev/ble_peripheral.dart';
+import 'package:blev/ble_socket.dart';
+
 import 'package:flutter/material.dart';
+import 'package:logger/logger.dart';
```

Comment:
I switched from `print`s that show up in the actual app to an actual Flutter logger. Reasoning being that in an eventual customer app, they probably won't want printing. If they want to share the content of some logs as actual UI messages, that should be their decision.

Reply:
yep makes sense